documentation/source/Policies/storage_policy.rst

```diff
@@ -0,0 +1,127 @@
+
+Storage Policies
+==================
+You can now define policies based on the type of application at the storage-class level. Following are some of the properties that can be customized at the default level in the openebs-storageclasses.yaml file.
+
+* jiva-replica-count
+* jiva-replica-image
+* jiva-controller-image
+* volume capacity and monitoring 
+
+Replica Count Policy
+----------------------
+You can specify the number of jiva replicas you want to create. In the following example, the jiva-replica-count is specified as 1. Hence, a single replica is created.  
+::
+
+    # Define a storage classes supported by OpenEBS
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+        name: openebs-standalone
+    provisioner: openebs.io/provisioner-iscsi
+    parameters:
+        pool: hostdir-var
+        openebs.io/jiva-replica-count: "1"
+    size: 5G
+
+Replica Image Policy
+----------------------
+You can add a replica image using the jiva-replica-image property. Specify the value in the "repo-hub-name/project-name:tag-name" format. In the following example, openebs is the repository name and jiva:0.4.0 is the project and tag name.
+::
+
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+        name: mysql
+    provisioner: openebs.io/provisioner-iscsi
+    parameters:
+        openebs.io/jiva-replica-count: "1"
+        openebs.io/capacity: "1G"
+        openebs.io/jiva-replica-image: "openebs/jiva:0.4.0"
+  
+Controller Image Policy
+---------------------------
+You can add a jiva controller image using the jiva-controller-image property. Specify the value in the ""repo-hub-name/project-name:tag-name" " format. In the following example, openebs is the repository name and jiva:0.4.0 is the project and tag name.
+::
+
+    apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+        name: mysql
+    provisioner: openebs.io/provisioner-iscsi
+    parameters:
+        openebs.io/jiva-replica-count: "1"
+        openebs.io/capacity: "1G"
+        openebs.io/jiva-controller-image: "openebs/jiva:0.4.0"
+
+Storage Pool Policy
+--------------------
+A storage pool provides a persistent path for an OpenEBS volume. It can be a directory on a
+
+* host-os or 
+* mounted disk
+
+You can specify key value pair in the openebs-config.yaml file. For example, key=openebs.io which is a storage-pool name and value=<*Name of storage pool CRD object*>
+
+::
+
+    # Define a storage pool for launching the replicas using
+    #  local directory or mounted directory from the minion nodes
+    #  
+    apiVersion: openebs.io/v1alpha1
+    kind: StoragePool
+    metadata:
+        name: sp-hostdir
+        type: hostdir
+    spec:
+        path: "/var/openebs" 
+
+Volume Capacity and Monitoring Policy
```

Comment:
These should not be joined together !!!
Keep Volume Capacity as different than Volume Monitoring Policy

Reply:
Let us remove the Capacity from storage policy.

Reply:
Let us remove the entire thing written below.

Instead rewrite as:
You can specify the volume monitoring policy using openebs.io/volume-monitor property.

Below is a sample Kubernetes StorageClass that makes use of volume monitoring policy
```
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: sc-percona-monitor
provisioner: openebs.io/provisioner-iscsi
parameters:
  openebs.io/jiva-replica-image: "openebs/jiva:0.4.0"
  openebs.io/volume-monitor: "true"
```

Reply:
Done